File: sol/1304/1304.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        if n == 2:
            return [1, -1]
        res = list(range(n - 1))
        res.append(-(n - 1) * (n - 2) // 2)
        return res

File: sol/1304/1304.cpp
struct Solution {
    vector<int> sumZero(int n) {
        if (n == 2){
            return {-1, 1};
        }
        vector<int> res(n);
        ranges::iota(res, 0);
        res[n - 1] = -(n - 1) * (n - 2) / 2;
        return res;
    }
};
